api/runtime/hooks/v1alpha1/inplaceupdate_types.go

```diff
@@ -20,7 +20,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 
-	clusterv1beta1 "sigs.k8s.io/cluster-api/api/core/v1beta1"
+	clusterv1 "sigs.k8s.io/cluster-api/api/core/v1beta2"
```

Comment:
When implementing my prototype I noticed that it would be better if we directly use v1beta2. It's not 100% consistent with our other hooks but v1beta2 is much better from a marshalling perspective and I don't want to run into trouble because of that

(also we can avoid down conversion in our controllers)

Reply:
I had a feeling Fabrizio mentioned we should use v1beta1 here

Reply:
Yes he did. We just realized later that it's not a good idea (we initially wanted it to use the same as in the other lifecycle hooks)